pom.xml

```diff
@@ -24,8 +24,8 @@
         <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
         <checkstyle-maven-plugin.version>3.2.0</checkstyle-maven-plugin.version>
         <cucumber.version>7.8.1</cucumber.version>
-        <jparest.version>0.1.11</jparest.version>
-        <kafka.commons.version>0.1.11</kafka.commons.version>
+        <jparest.version>0.1.14</jparest.version>
+        <kafka.commons.version>0.0.1</kafka.commons.version>
```

Comment:
I understand this is now a separate maven dependency, but I wonder if it would have been less confusing if the version was incremented from where it stopped when it was still a child of callisto-jparest module, rather than starting over at 0.0.1. 
If the above makes sense, you could go for 0.1.12 (as the latest version deployed on Artifactory was 0.1.11) or even increment the minor to mark the transition from Artifactory to Github Packages, i.e. 0.2.0

Reply:
Version updated to 0.2.0